update: extend communications framework

talzid opened the pull request:
# Description

Based on feedback from Glory, the communications guide has been extended to incorporate:
- feedback practices
- professional communication
- communication scenarios (late students, when to turn to UC or MD) - TBC

This has also introduced more files to the communications docs. Front page is simplified and links to other sources listed above.

Fixes #80 

## Type of change
- [x] Documentation (update or new)

# How Has This Been Tested?
Has be previewed in VS Code, unsure whether mermaid code will work

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have requested a review from Area Leads on the Pull Request

Commit:
ADD: fearless feedback and prof communication

Commit:
Add: split files and more content

more content to professional communication, and files split to simplify communication front page

Commit:
fix: run npm

tancnle commented on docs/communication/communication.md:
```diff
@@ -1,4 +1,6 @@
-# Communication
+# Communication <!-- omit in toc -->
+
+- [Guidelines](#guidelines)
```
**suggestion:** There is one section under this page hence I don't think TOC is needed. WDYT?

tancnle commented on docs/communication/communication.md:
```diff
@@ -13,26 +15,17 @@ are written down and accessible to everyone in the company.
+1. All announcements to Thoth Tech general channel only
+2. Use channels for all general to major discussion
+3. Use chat / direct message for private or personal discussion
+4. Use GitHub PRs for feedback on code or documentation
```
**nitpick:** According to the (draft) writing [guideline](https://github.com/thoth-tech/handbook/pull/75/files#diff-71243a89338f61a3a026b803479725353845d719f90cfce1b8ddd117764ff4efR146), we should use expanding version of acronyms on the first appearance. In fact, there is no need for acronyms in this case.

```suggestion
4. Use GitHub Pull Requests for feedback on code or documentation
```

tancnle commented on docs/communication/communication.md:
```diff
@@ -13,26 +15,17 @@ are written down and accessible to everyone in the company.
+- [Professional communication](docs/../professional-communication.md#professional-communication)
+- [Fearless feedback](docs/../fearless-feedback.md#fearless-feedback)
+- [Shoutouts](/docs/communication/fearless-feedback.md#shoutouts)
+- [Writing Style Guide](README.md) <!-- TBC -->
```
**suggestion:** There seems to be a mix of reference methods.

```suggestion
- [Professional communication](professional-communication.md)
- [Fearless feedback](fearless-feedback.md)
- [Shoutouts](fearless-feedback.md#shoutouts)
- Writing Style Guide
```

tancnle commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,87 @@
+
+**Will:** What will be done now? What are the next steps? Your opinion?
+
+It is important that you ask yourselves these questions. One should reflect and facilitate self-development and awarenessv before providing feedback.
```
**suggesiton:** Typo?

```suggestion
It is important that you ask yourselves these questions. One should reflect and facilitate self-development and awareness before providing feedback.
```

tancnle commented on docs/communication/professional-communication.md:
```diff
@@ -0,0 +1,71 @@
+- writing
+- speaking
+
+Effective communication differs from professional communication. For communication to be professional it is important that it is:
```
```suggestion
Effective communication differs from professional communication. For communication to be professional, it is important that it is:
```

tancnle commented on docs/communication/professional-communication.md:
```diff
@@ -0,0 +1,71 @@
+## Professional communication at Thoth Tech
+
+1. Use formal or semi-formal language when contacting someone for the first time and try to build rapport.
+2. Be respectful of a person's time. We do not expect you to be avaliable all the time and neither should your peers.If you want someones attention schedule meetings in advance or give a heads up.
```
```suggestion
2. Be respectful of a person's time. We do not expect you to be available all the time and neither should your peers. If you want someone's attention, schedule meetings in advance or give a heads up.
```

tancnle commented on docs/communication/professional-communication.md:
```diff
@@ -0,0 +1,71 @@
+8. Use Pull Requests or public chat channels instead of direct messages.
+9. Handle conflicts with diplomacy and an open-mind.
+10. Avoid controversial topics and gossip.
+11. Offer [praise](/docs/communication/fearless-feedback.md#shoutouts) and [fearless feedback](/docs/communication/fearless-feedback.md#fearless-feedback).
```
```suggestion
11. Offer [praise](fearless-feedback.md#shoutouts) and [fearless feedback](fearless-feedback.md#fearless-feedback).
```

tancnle commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,87 @@
+
+- Negative feedback should always be specific
+- General feedback for positive feedback is okay, but not ideal
+- Specifc feedback is the most impactful and the most productive
```
```suggestion
- Specific feedback is the most impactful and the most productive
```

tancnle commented on docs/communication/fearless-feedback.md:
````diff
@@ -0,0 +1,87 @@
+```mermaid
+  graph LR;
+  G[Goal] --> R[Reality] --> O[Options] --> W[Will]
+```
````
**suggestion:** This is great. Can we move this within the GROW model section below?

tancnle commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,87 @@
+
+## Feedback delivery
+
+When delivering feedback it is important to ask for permission beforehand. It is expected that some days are not going to be an appropriate times for feedback.
```
**suggestion:** I propose adding a clause as an example for when it is not a good time to give feedback.

```
Please make sure you deliver feedback when you are not upset. 
```

tancnle commented on docs/communication/professional-communication.md:
```diff
@@ -0,0 +1,71 @@
+
+1. Use formal or semi-formal language when contacting someone for the first time and try to build rapport.
+2. Be respectful of a person's time. We do not expect you to be avaliable all the time and neither should your peers.If you want someones attention schedule meetings in advance or give a heads up.
+3. All company data should be shareable by default.
```
**suggestion:** We can use `1.` for all items in the order list. Markdown will render them in the correct order. This will avoid human error and make inserting new items less noisy (ie. we do not need to increment numbers)

tancnle reviewed (commented):
Excellent work @talzid ❤️. I have left some suggestions for your consideration. Please let me know what you think 🙏🏼 .

Commit:
Update docs/communication/communication.md

Co-authored-by: Tan Le <tan.le@hey.com>

Commit:
Update docs/communication/communication.md

Co-authored-by: Tan Le <tan.le@hey.com>

Commit:
Update docs/communication/fearless-feedback.md

Co-authored-by: Tan Le <tan.le@hey.com>

Commit:
Update docs/communication/professional-communication.md

Co-authored-by: Tan Le <tan.le@hey.com>

Commit:
Update docs/communication/professional-communication.md

Co-authored-by: Tan Le <tan.le@hey.com>

Commit:
Update docs/communication/fearless-feedback.md

Co-authored-by: Tan Le <tan.le@hey.com>

Commit:
Update docs/communication/professional-communication.md

Co-authored-by: Tan Le <tan.le@hey.com>

Commit:
FIX: adding suggestions

Commit:
Merge branch '80-extend-communications-framework' of https://github.com/thoth-tech/handbook into docs/extend-communications-framework

tancnle commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,94 @@
+
+There are four different types of feedback that exist on this spectrum.
+
+|             |              |              |
+| :---------- | :----------: | -----------: |
+|             | **Positive** |
+| **General** |      +       | **Specific** |
+|             | **Negative** |              |
```
**praise:** This is a clever use of table format ❤️

tancnle commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,94 @@
+- Specific feedback is the most impactful and the most productive
+- Specifics are the best way to ensure that feedback is taken on-board and helpful
+
+## Feedback delivery
```
**suggestion:** It would be great to add a section about receiving feedback. What do you think?

Then we can name this 'Delivering feedback' and another section 'Receiving feedback'.

Commit:
update: add toc and moved content

Content moved to help with the flow. Can be difficult to tell the end of one sub section without formatting.

Commit:
ADD: Receiving feedback

Actioning @tancnle feedback (ironically)

Commit:
FIX: typos and word changes

Copy and pasting content to word. Using MS Editor to find errors and suggestions

Commit:
ADD: added scenarios file, update repo readme.md

Rename communication main file to fit standards (index.md). Updated README.md. Added last communication extension file and link in index.

ginge270 commented on README.md:
```diff
@@ -22,7 +22,7 @@ Please see the [contribution guidelines](CONTRIBUTING.md).
   - [Pitch](docs/marketing/pitch.md)
   - [Recruitment Ad](docs/marketing/recruitment-ad.md)
-- [Communication](docs/communication/communication.md)
+- [Communication](docs/communication/index.md)
```
Maybe "Communication Guide"

ginge270 commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,160 @@
+
+When delivering feedback, it is important to ask for permission beforehand. It is expected that some days are not going to be an appropriate time for feedback. Please do not deliver feedback when you or they are upset or angry. For feedback to be productive it is best to be level-headed. Willingness to listen and respect the feedback is a major part of the process. The outcome will not be product ...
+
+It can be as simple as a quick message: "_Is now an okay time for some feedback?_" or "_Would you be willing to hear some feedback from me?_".
```
another example is: Are you open to some feedback from me?

ginge270 commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,160 @@
+
+Feedback is used to help someone improve and learn, it is not a criticism of a person's skills or intelligence. Assuming positive intent, from both parties, in all feedback can reduce miscommunication or offence. However, this is a skill that takes time and practice.
+
+Particularly when providing constructive feedback, it is important to always seek to understand the root cause and context for a situation. Never assume to know everything about a person and make judgements. Feedback is designed to provide opportunities and create conversations for improvement. Being open to listen to the whole context can allow you to provide more specific and productive feedbac ...
```
*This is* particularly *important* when providing constructive

ginge270 replied:
Also, maybe bold the key phrases used through out like 'positive intent' and 'seeking to understand' so they stand out amongst the text blocks?

ginge270 commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,160 @@
+The GROW model is for deeper conversation around feedback. This framework naturally places feedback within the context of the team or project goals. It is also important that you ask yourself these questions. One should reflect and facilitate self-development and awareness before providing feedback.
+
+### Shoutouts
+
```
maybe link this as a way to build rapport?

ginge270 commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,160 @@
+
+Receiving feedback follows similar principles to delivery, however, in this case it is the response that changes.
+
+Mentally prepare to receive the feedback. Ideally, you should have given permission or asked for the feedback. Be patient, now is the time to acknowledge and listen to the advice given. Develop a growth mindset. This will help you step back from the defence and accept feedback to improve and learn.
```
Maybe add here to reflect on your mindset and ask yourself are you in a good place to receive feedbaack when being asked permision. It's okay to say now is not the right time, but what about tomorrow morning?

ginge270 commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,160 @@
+
+Mentally prepare to receive the feedback. Ideally, you should have given permission or asked for the feedback. Be patient, now is the time to acknowledge and listen to the advice given. Develop a growth mindset. This will help you step back from the defence and accept feedback to improve and learn.
+
+Again, you should always assume positive intent. Feedback is not a criticism on the work done, it is merely advice to improve and be even better next time around. Do not interrupt them as they are speaking. Listen to the whole story, rather than formulate challenges and excuses.
```
Another area that could be good to highlight in this guide is Active listening - and then link it into this section

ginge270 commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,160 @@
+- Assume positive intent
+- Seek to understand
+- Build rapport
+- Consider positioning
```
is positioning and framing the same thing?

ginge270 commented on docs/communication/fearless-feedback.md:
```diff
@@ -0,0 +1,160 @@
+**Act:** This stage is about creating actionable items derived from feedback. Not all feedback is actionable through outcomes or goals. Some feedback is about awareness and is highly situational. It is an important skill to learn how to action and assess feedback.
+
+The LADA model encourages the giver to freely convey their thoughts and you to dig deeper into the details of the feedback. By asking questions and going further you may uncover added information that might be valuable to your learning.
+
```
another feedback model is the AID model
Action, Impact, Do
https://andiroberts.com/aid-feedback-model/

tancnle reviewed (approved):
This is epic @talzid. Looks great to me. Let's merge and iterate.

Commit:
fix: changes based on feedback

ghost commented:
First iteration complete. Feedback taken and changes updated. 
Scenario file added with titles for guidance. Will be added to as Area Leads iterate for handover
